.editorconfig

```diff
@@ -9,10 +9,11 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
-[*.{sh,xml,launch}]
```

Comment:
Why remove xml and launch? This repo is for launch files

Reply:
Use spaces and indent of 4, included in the general case